src/components/UserProfile/UserProfileAccount/ProviderBalance/ChannelList/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Avatar from "@material-ui/core/Avatar";
+import { withStyles } from "@material-ui/styles";
+import map from "lodash/map";
+
+import { useStyles } from "./styles";
+import SingularityLogo from "../../../../../assets/images/avatar.png";
+import { cogsToAgi } from "../../../../../utility/PricingStrategy";
+
+const ChannelList = ({ classes, linkedProviders }) => {
+  return (
+    <React.Fragment>
+      <Grid xs={12} sm={12} md={12} lg={12} className={classes}>
+        <Grid xs={12} sm={12} md={12} lg={12} className={classes.columnTitle}>
+          <Grid xs={12} sm={12} md={7} lg={7} className={classes}>
+            <Typography>provider channels</Typography>
+          </Grid>
+          <Grid xs={12} sm={12} md={5} lg={5}>
+            <Typography>available tokens</Typography>
+          </Grid>
+        </Grid>
+        {map(linkedProviders, provider => {
+          const { org_name, hero_image, groups } = provider;
+          const firstGroup = groups[0];
+          const firstChannel = firstGroup.channels[0];
```

Comment:
we could handle `.channels of undefined` here in case firstGroup is empty

Reply:
Addressed in [commit](https://github.com/singnet/snet-dapp/pull/415/commits/b7e15cdc4d01eebe1f0c5fb62a175dbcded187c0)